app/admin/termin/page.js

```diff
@@ -39,75 +57,81 @@ function AdminTermin() {
     alert("PDF-Datei wird heruntergeladen.");
   };
 
-  const exampleEvents = [
-    { day: "Montag", time: "10:00", title: "Projektbesprechung" },
-    { day: "Dienstag", time: "14:00", title: "Meeting mit Kunden" },
-    { day: "Donnerstag", time: "09:00", title: "Schulung" },
-    { day: "Freitag", time: "15:00", title: "Team-Event" },
-  ];
-
-  const daysOfWeek = ["Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag", "Sonntag"];
+  const eventStyleGetter = (event) => {
+    let backgroundColor = "blue";
+    if (event.title.includes("Projekt")) backgroundColor = "green";
+    else if (event.title.includes("Meeting")) backgroundColor = "red";
+    else if (event.title.includes("Team")) backgroundColor = "orange";
+
+    return {
+      style: {
+        backgroundColor,
+        borderRadius: "5px",
+        color: "white",
+        padding: "5px",
+        border: "none",
+      },
+    };
+  };
 
   return (
     <StyledPaper>
       <DesignTitel> Willkommen im Admin-Terminmanagement </DesignTitel>
 
-      <Box sx={{ display: "flex", justifyContent: "center", alignItems: "center", mb: 3 }}>
-        <ToggleButtonGroup value={view} exclusive onChange={handleViewChange}>
-          <ToggleButton value="day">Tagesansicht</ToggleButton>
-          <ToggleButton value="month">Monatsansicht</ToggleButton>
-          <ToggleButton value="year">Jahresansicht</ToggleButton>
-        </ToggleButtonGroup>
+      <Box sx={{ flex: 1, display: "flex", justifyContent: "center", alignItems: "center", mb: 3 }}>
+        <Typography variant="h6">Kalenderübersicht</Typography>
       </Box>
 
-      <Box sx={{ flex: 1, display: "flex", justifyContent: "center", alignItems: "center" }}>
-        {view === "day" ? (
-          <Grid container spacing={2} justifyContent="center">
-            {daysOfWeek.map((day) => (
-              <Grid item xs={12} sm={6} md={4} lg={3} key={day}>
-                <StyledPaper>
-                  <Typography variant="h6" sx={{ marginBottom: 1 }}>
-                    {day}
-                  </Typography>
-                  <Box sx={{ flexGrow: 1 }}>
-                    {exampleEvents
-                      .filter((event) => event.day === day)
-                      .map((event, index) => (
-                        <Box key={index} sx={{ marginBottom: 1 }}>
-                          <Typography variant="body1">{event.time}</Typography>
-                          <Typography variant="body2">{event.title}</Typography>
-                        </Box>
-                      ))}
-                  </Box>
-                </StyledPaper>
-              </Grid>
-            ))}
-          </Grid>
-        ) : (
-          <Calendar view={view} onViewChange={setView} onClickDay={handleDateClick} />
-        )}
+      {/* React-Big-Calendar */}
```

Comment:
Auch hier bitte wieder auf die Sprache DE/EN achten. Da das anderes aussieht, als die Übersicht in der User-Ansicht, hier eine simple Komponente bauen?

Reply:
Warum kann der Admin nicht durch Klicken auf den Kalender ein neues Event erstellen, der User aber schon?

Reply:
durch klick event erstellen wir gefixt und die sprache.

Reply:
Sprache im Kalender immer noch Englisch/Deutsch gemischt, Events ohne Zeitangabe in der Monatsansicht, geht das hier vllt mit einem Subtext? Da ist ne Menge ungenutzter Platz 
![unten](https://github.com/user-attachments/assets/56605b55-c161-488a-9203-963bc1bd69dd), den man noch sinnvoll für den Kalender nutzen  kann